smooth-app-bar-layout/src/main/java/me/henrytao/smoothappbarlayout/SmoothAppBarLayout.java

```diff
@@ -151,6 +151,7 @@ public int getCurrentOffset() {
 
   public void setScrollTargetCallback(ScrollTargetCallback scrollTargetCallback) {
     mScrollTargetCallback = scrollTargetCallback;
+    setBehaviorScrollTargetCallback();
```

Comment:
Hi @dammitt, 

I double check the code. You are right. It's my mistake when migrating from v1 to v2. Thanks.